static/templates/view_code_in_playground.hbs

```diff
@@ -0,0 +1 @@
+<a target="_blank" class="code_external_link"  title="{{title}}" aria-label="{{title}}"><i class="fa fa-external-link" aria-hidden="true"></i></a>
```

Comment:
I don't think this should be aria-hidden - -that makes it inaccessible.  aria-hidden should be used only for elements that aren't clickable (E.g. the decoration icons we have next to text in some places)

Reply:
Also this needs `rel=noopener` type tags for security; look at any of our other links for an example.

Reply:
Interesting... Thanks!